engine/servermaster/jobmanager.go

```diff
@@ -613,3 +649,31 @@ func (jm *JobManagerImpl) WatchJobStatuses(
 	receiver := jm.notifier.NewReceiver()
 	return snapshot, receiver, nil
 }
+
+func (jm *JobManagerImpl) bgJobOperatorLoop(ctx context.Context) {
+	jm.wg.Go(func() error {
+		timer := time.NewTimer(jobOperateInterval)
+		for {
+			select {
+			case <-ctx.Done():
+				return errors.Trace(ctx.Err())
+			case <-jm.jobOperatorNotifier:
+			case <-timer.C:
```

Comment:
Why do we need both `jm.jobOperatorNotifier` and a timer?

Reply:
Changed to use `notify.Notifier`, which contains both manually trigger and periodic trigger